t/25-bugs.t

```diff
@@ -34,11 +34,11 @@ OpenQA::Test::Database->new->create(skip_fixtures => 1);
 my $t = Test::Mojo->new('OpenQA::WebAPI');
 
 
-my $bug = OpenQA::Schema::Result::Bugs->get_bug('poo#200', $t->app->db);
+my $bug = OpenQA::Schema::Result::Bugs::get_bug('poo#200', $t->app->db);
```

Comment:
In https://github.com/os-autoinst/openQA/pull/1999#pullrequestreview-212944806 I was told to use a helper function... perhaps we should agree on what style to go for?
@kraih Can you weigh in on that?

Reply:
Actually, this is something else since it's model code. Interactions with `DBIx::Class` are supposed to go through resultset objects. The fact that you have to pass `$t->app->db` here shows a bug. The correct version would look something like `my $bug = $schema->resultset('Bugs')->get_bug('poo#200');`.

Reply:
Actually you're right. That means the method should be moved from the result to the resultset.

@asdil12 I can change this for you if you like.